cloud/ec2_fleet.go

```diff
@@ -199,6 +199,25 @@ func (m *ec2FleetManager) GetInstanceStatus(ctx context.Context, h *host.Host) (
 	return status, nil
 }
 
+func (m *ec2FleetManager) CheckInstanceType(ctx context.Context, instanceType string) error {
```

Comment:
Could this be shared with ec2Manager (maybe in ec2_util.go)?

Reply:
I think sharing this isn't worth it: the only part that could really be shared without type conversions is lines 211 to 218. Most functions it looks like are duplicated between the ec2FleetManager and ec2Manager for this same reason so I'm personally okay with keeping this as is.

Reply:
sgtm